### ISSUE TITLE
User with many requests cannot be deleted

### ISSUE THREAD

**Issue by nmcmahon1215** [sct/overseerr#3322]
### Description

When deleting a user with a large number of a requests, the user deletion fails with an SQLite error.

Logs at the time of this request show: 

```
2023-02-06T01:16:56.284Z [error][API]: Something went wrong deleting a user {"userId":"4","errorMessage":"SQLITE_ERROR: Expression tree is too large (maximum depth 1000)"}
```

The user I was attempting to delete here had 1292 requests.

### Version

1.32.5

### Steps to Reproduce

1. Go to users
2. Find a user with a large number of requests (in my example 1292, but I suspect 1000 may be the limit based on the logging)
3. Click delete
4. UI displays "Something went wrong while deleting the user."
5. SQLite error logged: "Expression tree is too large (maximum depth 1000)"

### Logs

```shell
2023-02-06T01:22:39.679Z [error][API]: Something went wrong deleting a user {"userId":"4","errorMessage":"SQLITE_ERROR: Expression tree is too large (maximum depth 1000)"}
```

### Platform

desktop

### Device

Macbook Pro

### Operating System

macOS 13.2

### Browser

Safari

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by sct** [comment]
This is an issue with how TypeORM cascades deletions. We can manually delete associated requests before deleting a user to fix this.

**Comment by Zebebles** [comment]
Picking this up as a first issue

**Comment by OwsleyJr** [comment]
Fixed with #3376.
